firefox-ios/firefox-ios-tests/Tests/SyncIntegrationTestPlan.xctestplan

```diff
@@ -119,6 +119,9 @@
         "TabTrayTests",
         "TabTrayTests\/testAccessibility()",
         "TabsPerformanceTest",
+        "TabsTests",
+        "TabsTestsIpad",
+        "TabsTestsIphone",
```

Comment:
TopTabsTests wasn't excluded from SyncIntegrationTestPlan. It should not be run here.